### PR TITLE
Add 2020-05-14 – Knative Community Meetup #2

### PR DESCRIPTION
**Pull request by jonatasbaldin**
Add 2020-05-14 – Knative Community Meetup #2!

@macruzbar could u take a look? ✨

### CODE REVIEW

File: community/meetup/README.md [hunk 1]
```diff
@@ -10,6 +10,46 @@ Stay tuned for new events by subscribing to our [calendar](https://calendar.goog
 
 ---
 
+### 2020-05-14 – Knative Community Meetup #2
+
+Video: https://youtu.be/24owwOKj86E
+
+## Agenda
+- Welcome! (5’)
+  - Announce recording of meeting.
```
Comment (mattmoor) [inline community/meetup/README.md:19]: Remove trailing whitespace:
```suggestion
  - Announce recording of meeting.
```

File: community/meetup/README.md [hunk 2]
```diff
@@ -10,6 +10,46 @@ Stay tuned for new events by subscribing to our [calendar](https://calendar.goog
...
+  - Announce recording of meeting. 
+- Update from the Steering Committee (15’)
+  - TOC election results (Tomas Isdal)
+- Working groups updates (5’)
+- Demo - "Automating service delivery with bindings" by Evan Anderson, software engineer at VMware (30’)
+- Demo discussion / conversation (15’-20’)
+- Close (5’)
+- Take the [survey](https://docs.google.com/forms/d/e/1FAIpQLSebw2IOjmnStiUhPpnndpjyuBUoziZOw9PK9fnJeFBQX0QxWw/viewform)! (it’s good karma)
+
+## TOC Election results
```
Comment (mattmoor) [inline community/meetup/README.md:28]: Remove trailing whitespace:
```suggestion
## TOC Election results
```

File: community/meetup/README.md [hunk 3]
```diff
@@ -10,6 +10,46 @@ Stay tuned for new events by subscribing to our [calendar](https://calendar.goog
...
+- Close (5’)
+- Take the [survey](https://docs.google.com/forms/d/e/1FAIpQLSebw2IOjmnStiUhPpnndpjyuBUoziZOw9PK9fnJeFBQX0QxWw/viewform)! (it’s good karma)
+
+## TOC Election results 
+- Nghia Tran (Google) - new member
+- Markus Thömmes (Red Hat) - new member
+- Grant Rodgers (Google) - new member
+- Matt Moore (VMWare) - existing member
+- Evan Anderson (VMWare) - existing member
+
```
Comment (mattmoor) [inline community/meetup/README.md:34]: Remove trailing whitespace:
```suggestion

```

File: community/meetup/README.md [hunk 4]
```diff
@@ -10,6 +10,46 @@ Stay tuned for new events by subscribing to our [calendar](https://calendar.goog
...
+  - User facing docs: https://knative.dev/development/serving/configuring-autoscaling/
+  - Technical docs: https://github.com/knative/serving/blob/master/docs/scaling/SYSTEM.md
+- A lot of improvements to the loadbalancing in Knative has landed, vastly improving latency for concurrency-limited revisions. Give HEAD a whirl if you’re seeing issues there.
+- Speaking of issues: We need your input!
+  - We’re preparing a questionnaire to gather structured feedback regarding the types of workloads you’re running and which settings you find yourself tweaking to make autoscaling work in your favor.
+  - While we’re preparing that (will likely be sent out via the knative-users list), please feel free to give us free-form feedback on anything autoscaling. That can either be Github issues if you’re having issues, a thread on knative-users or send it to me privately if you can’t share publicly.
+
+## Eventing
+- Update on Brokers
+  - https://github.com/knative/eventing/issues/3139
```
Comment (mattmoor) [inline community/meetup/README.md:49]: Remove trailing whitespace:
```suggestion
  - https://github.com/knative/eventing/issues/3139
```

File: community/meetup/README.md [hunk 5]
```diff
@@ -10,6 +10,46 @@ Stay tuned for new events by subscribing to our [calendar](https://calendar.goog
...
+  - Announce recording of meeting.
+- Update from the Steering Committee (15’)
+  - TOC election results (Tomas Isdal)
+- Working groups updates (5’)
+- Demo - "Automating service delivery with bindings" by Evan Anderson, software engineer at VMware (30’)
+- Demo discussion / conversation (15’-20’)
+- Close (5’)
+- Take the [survey](https://docs.google.com/forms/d/e/1FAIpQLSebw2IOjmnStiUhPpnndpjyuBUoziZOw9PK9fnJeFBQX0QxWw/viewform)! (it’s good karma)
+
+## TOC Election results
```
Comment (mattmoor) [inline community/meetup/README.md:28]: Remove trailing whitespace:
```suggestion
## TOC Election results
```

File: community/meetup/README.md [hunk 6]
```diff
@@ -10,6 +10,46 @@ Stay tuned for new events by subscribing to our [calendar](https://calendar.goog
...
+- Nghia Tran (Google) - new member
+- Markus Thömmes (Red Hat) - new member
+- Grant Rodgers (Google) - new member
+- Matt Moore (VMWare) - existing member
+- Evan Anderson (VMWare) - existing member
+
```
Comment (mattmoor) [inline community/meetup/README.md:34]: Remove trailing whitespace:
```suggestion

```

File: community/meetup/README.md [hunk 7]
```diff
@@ -10,6 +10,46 @@ Stay tuned for new events by subscribing to our [calendar](https://calendar.goog
...
+  - User facing docs: https://knative.dev/development/serving/configuring-autoscaling/
+  - Technical docs: https://github.com/knative/serving/blob/master/docs/scaling/SYSTEM.md
+- A lot of improvements to the loadbalancing in Knative has landed, vastly improving latency for concurrency-limited revisions. Give HEAD a whirl if you’re seeing issues there.
+- Speaking of issues: We need your input!
+  - We’re preparing a questionnaire to gather structured feedback regarding the types of workloads you’re running and which settings you find yourself tweaking to make autoscaling work in your favor.
+  - While we’re preparing that (will likely be sent out via the knative-users list), please feel free to give us free-form feedback on anything autoscaling. That can either be Github issues if you’re having issues, a thread on knative-users or send it to me privately if you can’t share publicly.
+
+## Eventing
+- Update on Brokers
+  - https://github.com/knative/eventing/issues/3139
```
Comment (mattmoor) [inline community/meetup/README.md:49]: Remove trailing whitespace:
```suggestion
  - https://github.com/knative/eventing/issues/3139
```

File: community/meetup/README.md [hunk 8]
```diff
@@ -10,6 +10,45 @@ Stay tuned for new events by subscribing to our [calendar](https://calendar.goog
...
```
Comment (mattmoor) [inline community/meetup/README.md:48]: Remove trailing whitespace:
```suggestion
  - https://github.com/knative/eventing/issues/3139
```

**Comment by jonatasbaldin** [comment]
Git history gets all messed up using the GitHub "commit suggestion" 😅

**Comment by jonatasbaldin** [comment]
The failing tests are because other broken links... can we merge this nonetheless?  @macruzbar

**Comment by macruzbar** [comment]
> The failing tests are because other broken links... can we merge this nonetheless? @macruzbar

We can, we just need another `/lgtm` label
@abrennan89 @RichieEscarez , can you please approve?

**Comment by abrennan89** [comment]
/lgtm
/approve

Thanks @jonatasbaldin !

**Pull request merged**